pravega/pravega-client-python
# Update python bindings instructions

**Change log description**  
Updates the instructions to create Python bindings to `maturin` 0.14. This PR is a port from the rust client repo (see [#451](https://github.com/pravega/pravega-client-rust/pull/451)).

**Purpose of the change**  
Fixes [#450](https://github.com/pravega/pravega-client-rust/issues/450) (from the rust client repo).

**What the code does**  
Updated the README instructions.

**How to verify it**  
 Follow the updated steps in the README to create the bindings. Check that [pyproject.toml](https://github.com/pravega/pravega-client-python/blob/fe6818cb575f2b7fc5bb9c22baf87bce64c4139a/pyproject.toml#L2) requieres `maturin` 0.14.
